draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -350,9 +350,25 @@ control algorithms have the opportunity to improve the state of the art.
 
 ### Fairness within the Alternate Congestion Control Algorithm.
 
```

Comment:
I would change the title to:

1) reflect that what we need is really "do not starve"

2) align the "same algorithm" and "different algorithm" expectations.